src/tabs/tabs.spec.tsx

```diff
@@ -138,21 +113,16 @@ describe('Tabs', () => {
 
     window.requestAnimationFrame(() => {
```

Comment:
You are right to remove the done because that is no longer supported. What we should do instead is to make the async call into a promise and then return that promise in the test function. That way the test will wait for that promise to resolve before evaluating the expect.
That being said I have not tried that with the requestAnimationFrame before so I have no idea how jsdom is working with it.

Reply:
I updated this to return `new Promise` and resolve on pass